### ISSUE TITLE
How to enable prometheus flush metrics?

### ISSUE THREAD

**Issue by Fjsantucci** [SumoLogic/sumologic-kubernetes-collection#2740]
Hi all, I am using sumologic-kubernetes-collector version 2.14 deployed by helm.

I have enabled the metrics for fluentd/bit in the values.yaml file but I am not able to have all the existing metrics. The metric deployment is running as expected.
These are the output available metrics:

![image](https://user-images.githubusercontent.com/12071863/210111620-23876448-214b-4a09-8859-e321daf80d14.png)
![image](https://user-images.githubusercontent.com/12071863/210111680-737cab84-28db-4183-99a3-6ebbc5d0fd98.png)

Is there any existing documentation to enable for example: fluentd_output_status_flush_time_count , fluentd_output_status_emit_count or fluentd_output_status_rollback_count ?

Thanks.

**Comment by sumo-drosiek** [comment]
Hi @Fjsantucci 

After changes in fluentd, some metrics were not available: https://github.com/fluent/fluent-plugin-prometheus/pull/191

I see that the issue is fixed in https://github.com/fluent/fluent-plugin-prometheus/pull/197, so I'm going to prepare new sumologic fluentd image

**Comment by sumo-drosiek** [comment]
We are forwarding all metrics which are in Prometheus: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.14.1/deploy/helm/sumologic/values.yaml#L2212-L2217

Also we do not filter any incoming fluentd metrics. Please look what fluentd is actually exposing (please adjust command to your installation values):

```
kubectl port-forward -n "sumologic" service/collection-sumologic-fluentd-logs --address=0.0.0.0 24231
curl localhost:24231/metrics
```

**Comment by astencel-sumo** [comment]
Fixed in #503.

I can now see the metrics being scraped from Fluentd:

![image](https://user-images.githubusercontent.com/70892616/210339870-99cb5b10-1be2-4d51-a574-37ea030ec3c9.png)
